tests/testUserAccount.py

```diff
@@ -194,17 +192,18 @@ def test_user_can_login(self):
         self.assertEqual(response.status_code, 201)
         self.assertIn('token', response_data)
 
-    def test_user_can_login_with_email(self):
-        """test  user can log in with email"""
+    def test_user_cannot_login_with_invalid_password(self):
+        """test user cannot login with invalid details"""
         response = self.app.post('/api/v1/auth/login',
-                                 data=json.dumps(self.login_with_email),
+                                 data=json.dumps(self.invalid_password),
                                  content_type='application/json')
         response_data = json.loads(response.get_data().decode('utf-8'))
-        self.assertEqual(response.status_code, 201)
-        self.assertIn('token', response_data)
+        self.assertEqual(response.status_code, 401)
+        self.assertEqual(response_data['message'],
+                         'Invalid password.')
 
-    def test_user_cannot_login_with_invalid_password(self):
-        """test user cannot login with invalid details"""
+    def test_user_cannot_login_with_password_with_less_than_6_characters(self):
```

Comment:
This test is more appropriate during registration - I am assuming that you can only log in once you have registered.

Reply:
Yes, I get the point.Working on it now